src/mmw/apps/modeling/views.py

```diff
@@ -216,18 +218,15 @@ def _initiate_tr55_job_chain(model_input, job_id):
 def _construct_tr55_job_chain(model_input, job_id):
     job_chain = []
 
-    current_hash = model_input['modification_hash']
-    census = model_input.get('census')
-    census_hash = None
-
-    if census is not None:
-        census_hash = model_input['census'].get('modification_hash')
-
-    if census is None or current_hash != census_hash:
-        job_chain.append(tasks.prepare_census.s(model_input))
-        job_chain.append(tasks.run_tr55.s(model_input))
-    else:
-        job_chain.append(tasks.run_tr55.s(census, model_input))
+    # TODO put this into an if/else block and only do it if the
+    # censuses are not already cached.
+    aoi = model_input.get('area_of_interest')
+    pieces = model_input.get('modification_pieces')
+    polygons = [aoi] + [m['shape']['geometry'] for m in pieces]
+    job_chain.append(tasks.polygons_to_id.s(polygons))
+    job_chain.append(tasks.id_to_histogram.s())
+    job_chain.append(tasks.histograms_to_censuses.s())
+    job_chain.append(tasks.run_tr55.s(model_input))
```

Comment:
I am not sure of the exact reason, that this was done.  I think part of the reason is that they wanted visibility into the chain so that they could see what was in there in two of the unit tests.